app/src/main/java/org/mifos/selfserviceapp/ui/fragments/SavingAccountsTransactionFragment.java

```diff
@@ -202,6 +204,13 @@ public void resetFilter() {
                 setSavingAccountsTransactionList(transactionsList);
     }
 
+    @OnClick(R.id.tv_help_line_number)
+    void dialHelpLineNumber() {
+        Intent intent = new Intent(Intent.ACTION_DIAL);
+        intent.setData(Uri.parse("tel:" + getString(R.string.help_line_number)));
```

Comment:
This is not the part of this PR and It is already implemented in #219. Remove this and Please add screenshot for your changes in UI.